pages/about/faq.md

```diff
@@ -1,26 +1,30 @@
 ---
 title: FAQ
 layout: info
+description: Get answers to frequently asked questions about Lawrence Livermore National Laboratory's open source software.
 permalink: /about/faq/
 ---
 
-## {{ page.title }}
+# {{ page.title }}
 {: .page-header .no_toc}
 
-*These FAQs primarily target LLNL developers working in the [LLNL GitHub organization](https://github.com/LLNL). Don't see your question listed below? Please contact [LLNL GitHub admins](mailto:github-admin@llnl.gov).*
+<!-- START: Info box -->
+**These FAQs primarily target LLNL developers** working in the [LLNL GitHub organization](https://github.com/LLNL){% raw %}{% endraw %}. Don't see your question listed below? Please contact [LLNL GitHub admins](mailto:github-admin@llnl.gov).
+<!-- End: Info box -->
 
 * Table of Contents
 {:toc}
 
-### How do I set up a GitHub account?
+<!-- START: Accordions Each h2 below will be a separate accordion. If any of the requested functionality (e.g., info boxes) is a problem, let me know and I can adjust -->
+## How Do I Set up a GitHub Account?
 
-If you’re new to GitHub and open source in general, figuring out how to get set up can be a challenge. You may want to read through the GitHub Help pages on [setting up and managing your GitHub profile](https://help.github.com/categories/setting-up-and-managing-your-github-profile/).
+If you're new to GitHub and open source in general, figuring out how to get set up can be a challenge. You may want to read through the GitHub Help pages on [setting up and managing your GitHub profile](https://help.github.com/categories/setting-up-and-managing-your-github-profile/){% raw %}{% endraw %}.
```

Comment:
what's the rationale for `{% raw %}{% endraw %}.` ?

Reply:
Note that I see this in many places (every link?), and don't understand why it's there.